framework-test/src/main/java/org/checkerframework/framework/test/diagnostics/TestDiagnosticUtils.java

```diff
@@ -22,13 +23,14 @@ public class TestDiagnosticUtils {
 
     /** How the diagnostics appear in Java source files. */
     public static final String DIAGNOSTIC_IN_JAVA_REGEX =
-            "\\s*(error|fixable-error|warning|fixable-warning|other):\\s*(\\(?.*\\)?)\\s*";
+            "(?s)\\s*(error|fixable-error|warning|fixable-warning|other):\\s*(\\(?.*\\)?)\\s*";
```

Comment:
Can you add a comment, here and below, what this addition captures? (If possible, comments for the other parts would be great, too!) Same for the other change.

Reply:
I've added it.

Reply:
I forgot to change this dotall. I have updated it in the new commit.